### PR TITLE
Backend routine for submitting scripts

### LINKED ISSUE #16: Backend functionality for storing user created scripts
@viveksb007 @asitava1998 @CloudyPadmal In the [last PR](https://github.com/fossasia/pslab-remote/pull/14) , I embedded ace.js text editor on a page that allowed users to write their own Python code.

Now the question is how to store these.
Should they go into a database along with the Primary key user-email and a unique id for searchability, 
or
Should they be stored as text files in a directory.

Which begs the larger question ... if users are allowed to upload experiment results/screenshots as the app evolves, where is such data typically stored?

### PR DESCRIPTION
**Pull request by jithinbp**
scripts go into a new table along with the user-id for searchability and access control

fixes #16 .

### CODE REVIEW

File: app/views.py [hunk 1]
```diff
@@ -83,13 +83,38 @@ def logout():
...
 	else:
 		return render_template('error.html',error = 'Unauthorized Access')    
 
 
 
+@app.route('/addScript',methods=['POST'])
+def addScript():
+	try:
+		if session.get('user'):
+			_user = session.get('user')[1]
```
Comment (viveksb007) [inline app/views.py]: Could you also give ```_user``` as a parameter to addScript? Then in future we can make a call from Android or Desktop app to save script.

File: app/views.py [hunk 2]
```diff
@@ -83,13 +83,38 @@ def logout():
...
+				db.session.add(newSnippet)
+				db.session.commit()
+				return redirect('/userHome')
+			except Exception as exc:
+				return render_template('error.html',error = 'Write Failed.') 
+
+		else:
+			return render_template('error.html',error = 'Unauthorized Access')
+	except Exception as e:
+		print e
```
Comment (viveksb007) [inline app/views.py]: This is causing build error.

File: app/views.py (hunk 1 above)
Comment (jithinbp) [inline app/views.py]: The _user variable is automatically obtained from the session variable
`_user = session.get('user')[1] `
It is stored there when the user logs in, and if a login event has not occurred, an unauthorized access error will be raised. Did not make sense to allow a separate user field if the login event is necessary anyway. Is that okay?

File: app/views.py (hunk 2 above)
Comment (jithinbp) [inline app/views.py]: fixed

File: app/views.py (hunk 1 above)
Comment (viveksb007) [inline app/views.py]: Yeah was just saying that this restricts the use of this function to only remote app. If we send user parameter in function we can call this endpoint from any app. But android app hasn't developed to the point to save scripts, so we can change things later.

**Pull request merged**